irohad/main/impl/raw_block_loader.cpp

```diff
@@ -38,16 +33,7 @@ namespace iroha {
       return block_factory_.deserialize(document.value());
     }
 
-    void BlockInserter::applyToLedger(std::vector<model::Block> blocks) {
-      auto storage = factory_->createMutableStorage();
-      for (auto &&block : blocks) {
-        storage->apply(block, [](const auto &current_block, auto &query,
-                                 const auto &top_hash) { return true; });
-      }
-      factory_->commit(std::move(storage));
-    }
-
-    nonstd::optional<std::string> BlockInserter::loadFile(std::string path) {
+    nonstd::optional<std::string> BlockLoader::loadFile(std::string path) {
```

Comment:
Is there a case when nullopt can be returned? There should be either check on that file exists or function should return just string. IMO first option is better